.buildkite/pipeline-resource-definitions/kibana-gen-ai-daily.yml

```diff
@@ -0,0 +1,54 @@
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: bk-kibana-gen-ai-daily
+  description: Runs the GenAI FTR tests daily
+  links:
+    - url: 'https://buildkite.com/elastic/kibana-gen-ai-daily'
+      title: Pipeline link
```

Comment:
There are similar links in other pipeline defs, so I added one, but I have no idea if the link is going to be correct. Is there anything to manually do on builkite for that?

Reply:
I did not copy from the template, but from another pipeline def instead :shame:. Thanks

Reply:
Fixed in https://github.com/elastic/kibana/pull/203369/commits/869ecd5072dca9bd94ea94271b05fc775bf0ecaa